pkg/frmtr/tree.go

```diff
@@ -0,0 +1,84 @@
+package frmtr
+
+import (
+	"sort"
+	"strings"
+)
+
+type Files []string
+
+func (f Files) String() string {
+	root := &node{
+		name:     ".",
+		children: make(map[string]*node),
+	}
+
+	for _, path := range f {
+		parts := strings.Split(path, "/") // TODO:
```

Comment:
does this TODO mean anything?

Reply:
Yes, use relative path to the root for this.